CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Lava
+Thank you for considering contributing to Lava! We welcome contributions of all types, from code and documentation to design and outreach.
+
+Please note that all contributors are expected to follow the guidelines in our [CODE OF CONDUCT](CODE_OF_CONDUCT.md) file. This document sets out the standards of behavior expected from contributors to the project.
+
+## Types of Contributions
+Here are some ways you can contribute to Lava:
+
+* **Submit bug reports**. If you find a bug in Lava, you can submit an issue to the project's issue tracker. See format here: [ISSUE_TEMPLATE](ISSUE_TEMPLATE/ISSUE_TEMPLATE.md)
+
+* **Contribute code** (please see our [coding guidelines](#coding-guidelines) below). If you're a developer and want to contribute code to Lava, you can fork the repository and submit a pull request with your changes.
+
+* **Test and review pull requests.** Lava relies on the contributions of many people, and every pull request needs to be reviewed and tested before it can be merged. You can help by testing pull requests and providing feedback on their functionality and quality.
+
+* **Help with outreach and community building**. Lava is a community-driven project, and we rely on the participation of our users and contributors to succeed. You can help by spreading the word about Lava, answering questions on community forums, and helping to build a positive and inclusive community.
+
+
+## Getting Started
+If you're new to contributing to open source projects, we recommend checking out the [Open Source Guides](https://opensource.guide/) for an introduction to the process.
+
+To get started with contributing to Lava, familiarize yourself with the project by reading the project's [litepaper](https://lavanet.xyz/), Repo's README and documentation, trying out the project, and joining the project's Discord channels to ask questions and learn from other contributors.
+
+Support is detailed in [SUPPORT.md](SUPPORT.md)
+
+## Coding Guidelines
+Please follow these guidelines when contributing code to Lava:
+
```

Comment:
Consider adding a section or paragraph describing how to set up a local development environment, and how contributors can run tests locally

Reply:
Thanks @workingrush! for starters, this general contributing guidelines will be good enough to start. 
We'll probably add ad-hoc contributing guidelines for specific repo's.